### PR TITLE
[master] dec doc m

### PR DESCRIPTION
**Pull request by cmcmarrow**
### What does this PR do?
Add docs for label Pr

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [] Docs

### Commits signed with GPG?
Yes

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -522,11 +522,31 @@ this:
...
+   python -m nox -e "test-3(coverage=False)" -- --core-tests
+
+You can enable or disable test groups locally by passing their respected flag:
+
+* --no-fast-tests - Tests that are ~10s or faster. Fast tests make up ~75% of tests and can run in 10 to 20 minutes.
+* --slow-tests - Tests that are ~10s or slower.
+* --core-tests - Tests of any speed that test the root parts of salt.
+* --flaky-jail - Test that need to be temporarily skipped.
+
+In Your PR, you can enable or disable test groups by setting a label.
```
Comment (barbaricyawps) [inline CONTRIBUTING.rst:541]: Minor fix: "In Your PR" should be "In your PR."

File: doc/topics/tutorials/writing_tests.rst [hunk 2]
```diff
@@ -452,6 +452,69 @@ can be used
...
+            self.assertTrue(self.run_function("test.ping"))
+
+You can enable or disable test groups locally by passing there respected flag:
+
+* --no-fast-tests
+* --slow-tests
+* --core-tests
+* --flaky-jail
+
+In Your PR you can enable or disable test groups by setting a label.
```
Comment (barbaricyawps) [inline doc/topics/tutorials/writing_tests.rst:510]: Same minor error. Should be "In your PR."

**Review by barbaricyawps — DISMISSED** [review]
I'm approving so that I don't need to re-review, but just be aware there are a couple of typos.

**Review by s0undt3ch — APPROVED** [review]

**Comment by s0undt3ch** [comment]
This PR has conflicts?
The merge forward from 3006.x already includes these fixes?

**Review by barbaricyawps — APPROVED** [review]
Looks good! Approved.

**Pull request merged**